tests/admin/endpoints/test-class-enpoint-ryte.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
The filename has a typo, please change to `test-class-endpoint-ryte.php`.

The test results in the following error on Travis:
`Fatal error: Call to undefined method WPSEO_Endpoint_Ryte_Test::createMock() in /tmp/wordpress/src/wp-content/plugins/wordpress-seo/tests/admin/endpoints/test-class-enpoint-ryte.php on line 16`